homeassistant/components/device_tracker/bluetooth_le_tracker.py

```diff
@@ -19,6 +21,12 @@
 
 BLE_PREFIX = 'BLE_'
 MIN_SEEN_NEW = 5
+CONF_SCAN_DURATION = "scan_duration"
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Optional(CONF_SCAN_DURATION, default=10): vol.All(vol.Coerce(int),
```

Comment:
There is no "postive_integer" method in Home assistants config validation module or do I miss something?